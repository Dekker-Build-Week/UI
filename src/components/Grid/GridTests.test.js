import React from 'react';

import GridContainer from './GridContainer';
import GridItem from './GridItem';
import renderer from 'react-test-renderer';

describe('Grid test suite', () => {
    it('Renders the GridContainer correctly', () => {
        const tree = renderer
            .create(<GridContainer></GridContainer>)
            .toJSON();

        expect(tree).toMatchSnapshot();
    }),
    it('Renders the GridItem correctly', () => {
        const tree = renderer
            .create(<GridItem></GridItem>)
            .toJSON();
        
        expect(tree).toMatchSnapshot();
    }),
    it('Renders a GridContainer with GridItems correctly', () => {
        const tree = renderer 
            .create(<GridContainer>
                <GridItem></GridItem>
                </GridContainer>)
                .toJSON();

        expect(tree).toMatchSnapshot(); 
    })
});