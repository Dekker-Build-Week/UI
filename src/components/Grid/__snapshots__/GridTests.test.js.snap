// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid test suite Renders a GridContainer with GridItems correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-grid-1 MuiGrid-container"
>
  <div
    className="MuiGrid-root makeStyles-grid-105 MuiGrid-item"
  />
</div>
`;

exports[`Grid test suite Renders the GridContainer correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-grid-1 MuiGrid-container"
/>
`;

exports[`Grid test suite Renders the GridItem correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-grid-105 MuiGrid-item"
/>
`;
